staging/src/k8s.io/apiserver/pkg/server/config_test.go

```diff
@@ -179,6 +181,27 @@ func TestNewWithDelegate(t *testing.T) {
 		"/readyz/shutdown",
 	}
 	checkExpectedPathsAtRoot(server.URL, expectedPaths, t)
+
+	// wait for health (max-in-flight-filter is initialized asynchronously, can take a few milliseconds to initialize)
+	if err := wait.PollImmediate(100*time.Millisecond, wait.ForeverTestTimeout, func() (bool, error) {
+		// healthz checks are installed in PrepareRun
+		resp, err := http.Get(server.URL + "/healthz")
```

Comment:
I think this can be simplified to exclude the checks we expect to fail and waiting for everything else to be healthy:

`/healthz?exclude=wrapping-health&exclude=delegate-health`